bootstrap.sh

```diff
@@ -4,9 +4,18 @@
 git submodule init
 git submodule update
 
-# install eigen, keyutils, kerberos, sparsehash, gflags, glog
+# install eigen, keyutils, kerberos, sparsehash, gflags, glog, lz4
 cd third_party
 
+if [ ! -d "lz4-dev" ]; then
+  wget https://github.com/lz4/lz4/archive/dev.zip
+  unzip dev.zip
+fi
+cd lz4-dev
+make
+sudo make install
```

Comment:
We should avoid using sudo here because that requires admin permissions.

I don't think you need this command, because when you do make install you install the libraries in some system directory, but we don't use that. We use the library in the lz4-dev directory that you wget.

Can you check?